libnavui-maps/src/main/res/drawable/mapbox_bg_button.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@abhishek1508 I am extremely confused here. The new buttons are rectangular with 10px border radius. https://www.figma.com/file/6bXLQGCUaYb7DdJsNL3e0Z/DS-Navigation?node-id=3334%3A39612
Am I working with wrong designs?

Reply:
No you are correct that they are rectangular by design. All I am saying is that we should make them that way in the DropInUI by providing a different background. Since this PR is adding a standalone UI component, to me it makes sense to have them look the same as other floating buttons; in this case circular. Later, in DropInUI module we can use this rectangular background and apply it to all the buttons

Reply:
Can you send me a link to Figma designs I should use?